docs/pages/router/advanced/anchor-routes.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Anchor Routes
+description: Learn how to use the anchor routes to modify navigation history
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+Anchor routes, sometimes referred to as initial routes, are routes that are injected into the start of the navigation history when navigator is loaded.
+
+Expo Router uses a different term to React Navigation due to the different philosophy of the API design. 
```

Comment:
```suggestion
```

Reply:
move this to later, also update the migration guide to mention the anchor stuff.